halo2_proofs/src/plonk/circuit.rs

```diff
@@ -376,6 +441,10 @@ pub trait Assignment<F: Field> {
         to: Option<Assigned<F>>,
     ) -> Result<(), Error>;
 
+    /// Get challenge value. If current phase is still earlier than requested one, a `None`
+    /// will be returned.
+    fn get_challenge(&mut self, challenge: Challenge) -> Option<F>;
```

Comment:
This needs a changelog entry.

Reply:
Added in [`7786bf6`](https://github.com/zcash/halo2/pull/593/commits/7786bf60ec4641bf5593938a87f46048a79d2a97).